backend/app/models/inquiry.py

```diff
@@ -1,27 +1,33 @@
 import uuid
 from datetime import datetime, timezone
+from typing import List, Optional
 
-from sqlmodel import Field, SQLModel
+from sqlmodel import Field, Relationship, SQLModel
+
+
+class Response(SQLModel, table=True):
+    id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True)
+    content: str = Field(min_length=1, max_length=255)
```

Comment:
Hmm was this in the DB schema for an inquiry response? Did we define max length for responses? (I know we have freeform responses in P2, not sure it's worth having that now.

Reply:
I thought we agreed on 256 or 512

Reply:
that's for inquiries. Responses (right now) don't have a text field at all, and we hadn't talked (iirc) about limiting it when we do.

Reply:
refactored and addressed